### PR TITLE
Feature/collection/collection ui

### PR DESCRIPTION
**Pull request by Creatorbon**

### CODE REVIEW

**Review by ivanpatsaibinary — CHANGES_REQUESTED** [review]

File: client/src/components/CollectionModal/modals/AddCollectionBody.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+          name: formData.name || editableCollection.name,
+          description: formData.description || editableCollection.description,
+        }
+      : formData,
+
+    validationSchema,
+
+    onSubmit: (values, { setSubmitting }) => {
+      setSubmitting(false);
+      setFormData(values);
```
[the remark this answers is not in the document]
Comment (Creatorbon) [reply client/src/components/CollectionModal/modals/AddCollectionBody.js:32]: setSubmitting - standart formik function. In docs, they recommend to use it. [docs](https://formik.org/docs/api/formik#setsubmitting-issubmitting-boolean--void)

setFormData - action for saving data, from an edit form. And if we get an error from the server, we don't need to fill the form again

File: client/src/components/CollectionList/CollectionListItem.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import CollectionsIcon from '@material-ui/icons/Collections';
+import StarIcon from '@material-ui/icons/Star';
+import { useStyles } from './styles';
+
+const CollectionListItem = ({ name }) => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.collectionItem}>
+      {name === 'My personal collection' ? <StarIcon /> : <CollectionsIcon />}
```
[the remark this answers is not in the document]
Comment (Creatorbon) [reply client/src/components/CollectionList/CollectionListItem.js:12]: I will make global constant

File: client/src/components/CollectionModal/modals/AddCollectionBody.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+const AddUserBody = ({ closeModal, editableCollection, setFormData, formData, addCollection, updateCollection }) => {
+  const classes = useStyles();
+
+  const formik = useFormik({
+    initialValues: editableCollection
+      ? {
+          name: formData.name || editableCollection.name,
+          description: formData.description || editableCollection.description,
+        }
+      : formData,
```
[the remark this answers is not in the document]
Comment (Creatorbon) [reply client/src/components/CollectionModal/modals/AddCollectionBody.js]: done

File: client/src/components/CollectionList/CollectionListItem.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Creatorbon) [reply client/src/components/CollectionList/CollectionListItem.js]: done

**Review by YuriiStetsula — APPROVED** [review]

**Review by ivanpatsaibinary — APPROVED** [review]

**Pull request merged**